transcendence-app/src/authorization/casl-ability.factory.ts

```diff
@@ -96,9 +98,17 @@ export class CaslAbilityFactory {
 
       can(Action.Create, Chatroom);
       can(Action.Read, Chatroom);
-      if (user.crm_member && user.crm_owner) {
-        can(Action.Update, Chatroom);
-        can(Action.Delete, Chatroom);
+      if (user.crm_member) {
+        if (!user.crm_banned) {
+          can(Action.Join, Chatroom, { id: user.chatId });
```

Comment:
this is redundant. If user.crm_member, then {id: user.chatId} always. But I think it's somehow clearer like this. Let me know if you disagree